src/Dodo.HttpClient.ResiliencePolicies/RetrySettings/JitterRetrySettings.cs

```diff
@@ -8,45 +8,29 @@ namespace Dodo.HttpClientResiliencePolicies.RetrySettings
 {
 	public class JitterRetrySettings : IRetrySettings
 	{
-		public int RetryCount { get; }
-		public TimeSpan MedianFirstRetryDelay { get; }
-		public Func<int, TimeSpan> SleepDurationProvider { get; }
-		public Action<DelegateResult<HttpResponseMessage>, TimeSpan> OnRetry { get; set; }
-
-		public JitterRetrySettings(int retryCount) : this(retryCount, _defaultMedianFirstRetryDelay)
+		public JitterRetrySettings()
 		{
-		}
+			RetryCount = Defaults.Retry.RetryCount;
+			SleepDurationProvider = _defaultSleepDurationProvider(RetryCount, _defaultMedianFirstRetryDelay);
 
-		public JitterRetrySettings(int retryCount, Action<DelegateResult<HttpResponseMessage>, TimeSpan> onRetry) :
-			this(retryCount, _defaultMedianFirstRetryDelay, onRetry)
-		{
+			OnRetry = _doNothingOnRetry;
 		}
 
-		public JitterRetrySettings(int retryCount, TimeSpan medianFirstRetryDelay) : this(retryCount,
-			medianFirstRetryDelay, _defaultOnRetry)
-		{
-		}
+		public int RetryCount { get; set; }
 
-		public JitterRetrySettings(
-			int retryCount,
-			TimeSpan medianFirstRetryDelay,
-			Action<DelegateResult<HttpResponseMessage>, TimeSpan> onRetry)
-		{
-			RetryCount = retryCount;
-			SleepDurationProvider = _defaultSleepDurationProvider(retryCount, medianFirstRetryDelay);
-			OnRetry = onRetry;
-		}
+		public Func<int, TimeSpan> SleepDurationProvider { get; set; }
 
-		public static IRetrySettings Default() => new JitterRetrySettings(Defaults.Retry.RetryCount);
+		public Action<DelegateResult<HttpResponseMessage>, TimeSpan> OnRetry { get; set; }
 
 		private static readonly TimeSpan _defaultMedianFirstRetryDelay =
 			TimeSpan.FromMilliseconds(Defaults.Retry.MedianFirstRetryDelayInMilliseconds);
 
 		// i - retry attempt
-		private static readonly Func<int, TimeSpan, Func<int, TimeSpan>> _defaultSleepDurationProvider =
+		//todo bulanova: временно открыла для тестов! после какой-нибудь реализации SleepDurationProvider проверить
```

Comment:
1. Could you write TODO in English please. Just to be consistent with language in the whole repository.
2. Do we really need to do tests this way? See comments in the test files.